src/config/plugins.ts

```diff
@@ -41,5 +42,7 @@ export const generatePlugins = (options: {
         successComment: false,
       },
     ],
-  ] as PluginSpec[];
+  ].filter(
+    (value: Array<any> | string): boolean => value.length > 0,
+  ) as PluginSpec[];
```

Comment:
It seems to be much simpler to add it conditionally using an array spread, the same way as it's done for other plugins?

Reply:
The npm plugin is being added as a `string` while the other plugin is an `array`. The intention was to keep the same types and not to return empty elements of `array` or `string`.

Reply:
Using array spread instead.